storage/innobase/include/univ.i

```diff
@@ -521,6 +521,9 @@ computers! */
 
 #define UNIV_SQL_NULL ULINT32_UNDEFINED
 
+/** Flag for default value of instant added column */
+#define UNIV_SQL_DEFAULT (UNIV_SQL_NULL - 1)
+
```

Comment:
Could we make `UNIV_SQL_NULL` an alias to `ULINT_UNDEFINED`, so that will not introduce a compatibility issue on 64-bit systems? It is not completely unthinkable to store a `(1<<32)-2`-byte BLOB into the database.